test/integration/verify/controls/aws_rds_cluster.rb

```diff
@@ -0,0 +1,38 @@
+title 'Test single AWS Aurora Cluster'
+
+aws_rds_cluster_identifier = attribute(:aws_rds_cluster_identifier, default: '', description: 'The AWS RDS Cluster identifier.')
+aws_rds_cluster_database_name = attribute(:aws_rds_cluster_database_name, default: '', description: 'The AWS RDS Cluster name.')
+aws_rds_cluster_engine = attribute(:aws_rds_cluster_engine, default: '', description: 'The AWS RDS Cluster engine.')
+aws_rds_cluster_engine_version = attribute(:aws_rds_cluster_engine_version, default: '', description: 'The AWS RDS Cluster engine version.')
+aws_rds_cluster_master_user = attribute(:aws_rds_cluster_master_user, default: '', description: 'The AWS RDS Cluster username.')
+
+
+control 'aws-rds-cluster-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS RDS Cluster has the correct properties.'
+
+  describe aws_rds_cluster(db_cluster_identifier: aws_rds_cluster_identifier) do
+    it { should exist }
+    it { should have_encrypted_storage }
+    its ('storage_encrypted') { should eq true }
```

Comment:
please use unified style: 
its('database_name')
instead of 
its ('database_name')